### DISCUSSION TITLE
Chizhik TV

### DISCUSSION THREAD

**Discussion by SuperWillow** [iptv-org/community#288] [Q&A]
### Channel ID

Chizhik.ru

### Stream URL

??? (Адрес плейлиста неизвестен из-за Триколор Онлайн ТВ, поэтому сложно его найти)

### Notes

Добавьте плейлист для просмотра телеканала Чижик, хотя-бы. А то можно просмотреть только на Триколоре, можно хотя-бы его добавить?

(translated with DeepL)
Add a playlist for watching TV channel Chizhik, at least. And you can only watch it on Tricolor, can you at least add it?

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Paid channel, required login to watch. Only available through Tricolor.tv

**Comment by BellezaEmporium** [comment]
> > Paid channel, required login to watch. Only available through Tricolor.tv
> 
> Перевод с Google Translator: Платный канал, для просмотра требуется вход. Доступно только через Триколор.
> 
> Печально 😞. Можно было добавить только плейлист с самим телеканалом из m3u плейлиста Триколора и телепрограмму хотя-бы, как у Рыжего?
>
> (translated with DeepL)
> Sad 😞. You could only add a playlist with the TV channel itself from the m3u playlist of Tricolor and a TV program at least, like Red?

If we manage to get access to Tricolor channels, maybe !

**Comment by BellezaEmporium** [comment]
> > > > Paid channel, required login to watch. Only available through Tricolor.tv
> > > 
> > > 
> > > Перевод с Google Translator: Платный канал, для просмотра требуется вход. Доступно только через Триколор.
> > > Печально 😞. Можно было добавить только плейлист с самим телеканалом из m3u плейлиста Триколора и телепрограмму хотя-бы, как у Рыжего?
> > > (translated with DeepL)
> > > Sad 😞. You could only add a playlist with the TV channel itself from the m3u playlist of Tricolor and a TV program at least, like Рыжий?
> > 
> > 
> > If we manage to get access to Tricolor channels, maybe !
> 
> Ладно, мне кажется на странице телеканала, есть пакет Комбо и кнопка, чтобы смотреть телеканал. Поскольку, телеканал платный и можно добавить к нему бесплатный IPTV плейлист, чтобы его смотреть. @LaneSh4d0w, как тебе идея? А то я не могу говорить по английски:( ![chrome_screenshot_1668719426398](https://user-images.githubusercontent.com/87235362/204099600-b00205f9-f624-4484-b3d3-211e502aeb1f.png)
> 
> (translated with DeepL)
>
> Okay, I think on the TV channel page, there is a Combo package and a button to watch the TV channel. Since, the TV channel is paid and you can add a free IPTV playlist to it to watch it. @LaneSh4d0w, what do you think of the idea? Because I can't speak English:(
>

Sadly I can't speak Russian too. @freearhey if you have an idea over this, please let me know.

**Comment by BellezaEmporium** [comment]
> @LaneSh4d0w, Okay, I think on the TV channel page, there is a Combo package and a button to watch the TV channel. Since, the TV channel is paid and you can add a free IPTV playlist to it to watch it.

If they somehow keep their TV stream link open for weird reasons I can grab it after logging myself in, but if I need to pay for it it's a dead end.

**Comment by kaikyoofers** [comment]
> > > @LaneSh4d0w, Okay, I think on the TV channel page, there is a Combo package and a button to watch the TV channel. Since, the TV channel is paid and you can add a free IPTV playlist to it to watch it.
> > 
> > 
> > If they somehow keep their TV stream link open for weird reasons I can grab it after logging myself in, but if I need to pay for it it's a dead end.
> 
> @LaneSh4d0w, попробуй покопаться здесь: https://www.google.com/url?sa=t&source=web&rct=j&url=https://satellitetvforum.info/viewtopic.php%3Ft%3D9409&ved=2ahUKEwjD7MrTs8z7AhUMmYsKHSqPCrcQFnoECBgQAQ&usg=AOvVaw3Ys7irlbZyfLe_rmCgS6T5 Может это поможет, а если нет бери с сайта Триколора плейлист телеканала Чижик.

я получаю ошибку контрольной суммы


i get a checksum error

**Comment by kaikyoofers** [comment]
> > > > > @LaneSh4d0w, Okay, I think on the TV channel page, there is a Combo package and a button to watch the TV channel. Since, the TV channel is paid and you can add a free IPTV playlist to it to watch it.
> > > > 
> > > > 
> > > > If they somehow keep their TV stream link open for weird reasons I can grab it after logging myself in, but if I need to pay for it it's a dead end.
> > > 
> > > 
> > > @LaneSh4d0w, попробуй покопаться здесь: https://www.google.com/url?sa=t&source=web&rct=j&url=https://satellitetvforum.info/viewtopic.php%3Ft%3D9409&ved=2ahUKEwjD7MrTs8z7AhUMmYsKHSqPCrcQFnoECBgQAQ&usg=AOvVaw3Ys7irlbZyfLe_rmCgS6T5 Может это поможет, а если нет бери с сайта Триколора плейлист телеканала Чижик.
> > 
> > 
> > я получаю ошибку контрольной суммы
> > i get a checksum error
> 
> Нет. Бери тогда с сайта Триколора, это поможет:)

проблема. я не живу в россии




problem. i dont live in russia

**Comment by BellezaEmporium** [comment]
have fun : https://limehd.tv/api/v4/channel/chizhikhd

for other channels, play with this API : https://limehd.tv/api/v4/playlist?page=8&limit=37&onlyavailable=0&epg=1&epgcnt=0&tz=3 and get the channel you want by putting the "address" name at the end of the "/channel/" link.
